backend/capellacollab/projects/toolmodels/modelsources/git/handler/handler.py

```diff
@@ -76,7 +76,7 @@ async def get_file_from_repository_or_artifacts_as_json(
         trusted_file_path: str,
         job_name: str,
         revision: str | None = None,
-    ) -> tuple[datetime.datetime, dict[str, t.Any]]:
+    ) -> tuple[datetime.datetime, list[dict[str, str | bool]]]:
```

Comment:
This was an error in the type annotation? (it's not obvious from the code)

Reply:
Yes. Its already some time ago that I did this PR but I think the type annotation here corresponds to json a file that look like this [coffee machine index.json](https://github.com/DSD-DBS/coffee-machine/blob/diagram-cache/main/diagram_cache/index.json).

Reply:
A typed dict would be even better.

Reply:
Certainly, but this first requires https://github.com/DSD-DBS/py-capellambse/pull/342 to be merged